omnibus/package-scripts/agent/postinst

```diff
@@ -68,9 +68,9 @@ if [ "$DISTRIBUTION" != "Darwin" ]; then
         # Only supports systemd and upstart
         echo "Enabling service $SERVICE_NAME"
         if command -v systemctl >/dev/null 2>&1; then
-            systemctl enable $SERVICE_NAME
-            systemctl enable $SERVICE_NAME-process
-            systemctl enable $SERVICE_NAME-trace
+            systemctl enable $SERVICE_NAME || true
```

Comment:
We should at least display a warning, like when neither systemctl nor initctl are found.